packages/manager/modules/adp/src/adp.component.js

```diff
@@ -0,0 +1,35 @@
+import angular from 'angular';
+import 'ovh-ui-kit/dist/oui.css';
+
+import './adp.scss';
+import template from './adp.html';
+import deploy from './deploy';
+import ADPService from './adp.service';
+import ADPConstants from './adp.constants';
+import controller from './adp.controller';
+
+const moduleName = 'ovhManagerAdpComponent';
+
+angular
+  .module(moduleName, [deploy])
+  .config(($stateProvider) => {
+    $stateProvider.state('adp', {
+      url: '/adp',
+      component: 'ovhManagerAdpComponent',
+      translations: {
+        value: [
+          '.',
+        ],
+        format: 'json',
+      },
+    });
+  })
+  .component('ovhManagerAdpComponent', {
+    template,
+    controller,
+    controllerAs: '$ctrl',
+  })
+  .service('AdpService', ADPService)
```

Comment:
Prefer using camelCase for services:

```suggestion
  .service('adpService', ADPService)
```